### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by SparkYuan**
update readme

### CODE REVIEW

**Comment by Peefy** [comment]
How about adding links to sub-projects like KCLVM, KCLOpenAPI, etc.?

**Comment by ldxdl** [comment]
LGTM

**Review by ldxdl — APPROVED** [review]

**Review by chai2010 — APPROVED** [review]
LGTM

File: README-zh.md [hunk 1]
```diff
@@ -1,73 +1,67 @@
...
 
 [![GitHub release](https://img.shields.io/github/release/KusionStack/kusion.svg)](https://github.com/KusionStack/kusion/releases)
 [![Go Report Card](https://goreportcard.com/badge/github.com/KusionStack/kusion)](https://goreportcard.com/report/github.com/KusionStack/kusion)
 [![license](https://img.shields.io/github/license/KusionStack/kusion.svg)](https://github.com/KusionStack/kusion/blob/main/LICENSE)
 [![Go Reference](https://pkg.go.dev/badge/github.com/KusionStack/kusion.svg)](https://pkg.go.dev/github.com/KusionStack/kusion)
 [![Coverage Status](https://coveralls.io/repos/github/KusionStack/kusion/badge.svg)](https://coveralls.io/github/KusionStack/kusion)
+</div>
+
+# Kusion
+Kusion 是 [KusionStack](https://github.com/KusionStack) 的引擎，用于解析用户在 [Konfig](https://github.com/KusionStack/konfig) 中描述的运维意图，并根据这些运维意图对真实的基础设执行相应的操作
```
[the remark this answers is not in the document]
Comment (SparkYuan) [reply README-zh.md:20]: 后面不再区分这两个东西了，都是kusion，作为一个整体看

File: README-zh.md [hunk 2]
```diff
@@ -1,73 +1,67 @@
...
+## 核心能力
 
-> KusionStack 提供面向云原生生态的定义及最佳实践，提供静态类型配置语言及工具支持，在业务镜像外提供 **Compile to Cloud** 的技术栈。Kusion 使用 Golang 语言编写，并具有跨 Unix-like 平台属性。
+- **应用全生命周期管理**: 结合 [Kusion](https://github.com/KusionStack/kusion) 与 [Konfig](https://github.com/KusionStack/konfig) 实现从应用第一行配置代码到生产可用的全生命周期管理
+- **多层级管理**: 原生支持多租户、多环境运维能力
+- **混合运行时**: 以统一的方式运维 Kubernetes 和 Terraform 等多种运行时的资源
+- **厂商无关**: 一次编写，动态渲染，多云运行
```
[the remark this answers is not in the document]
Comment (SparkYuan) [reply README-zh.md:26]: ditto

File: README-zh.md [hunk 3]
```diff
@@ -1,73 +1,67 @@
...
-make build-all
-
-# Build kusion & kcl tool chain for macOS
-# make build-local-darwin-all
-# Build kusion & kcl tool chain for linux
-# make build-local-linux-all
-# Build kusion & kcl tool chain for windows
-# make build-local-windows-all
+# cURL
+curl -sSf https://raw.githubusercontent.com/KusionStack/kusionup/main/scripts/install.sh | bash
```
[the remark this answers is not in the document]
Comment (SparkYuan) [reply README-zh.md:58]: added

File: README-zh.md [hunk 4]
```diff
@@ -1,73 +1,67 @@
-## 简介
+<div align="center">
+<p></p><p></p>
+<p>
+    <img  src="docs/logo.png">
+</p>
+<h1>A Unified Programmable Configuration Tech Stack</h1>
+
+[简体中文](https://github.com/KusionStack/kusion/blob/main/README-zh.md) | [English](https://github.com/KusionStack/kusion/blob/main/README.md) | [Konfig](https://github.com/KusionStack/konfig) | [KCLVM](https://github.com/KusionStack/KCLVM) | [Kusion](https://github.com/KusionStack/kusion) | [https://kusionstack.io](https://kusionstack.io/)
```
[the remark this answers is not in the document]
Comment (SparkYuan) [reply README-zh.md]: fixed

**Review by elliotxx — APPROVED** [review]
lgtm

**Review by chai2010 — APPROVED** [review]

**Pull request merged**